src/txdb.h

```diff
@@ -75,11 +73,13 @@ class CCoinsViewDB : public CCoinsView
 public:
     CCoinsViewDB(size_t nCacheSize, bool fMemory = false, bool fWipe = false);
 
-    bool GetCoins(const uint256 &txid, CCoins &coins) const;
-    bool HaveCoins(const uint256 &txid) const;
+    bool GetCoins(const COutPoint &outpoint, Coin &coin) const;
```

Comment:
nit: while you're changing these lines, it'd be nice to add overrides where appropriate.